Resources/Locale/en-US/discord/watchlist-connections.ftl

```diff
@@ -0,0 +1,17 @@
+discord-watchlist-connection-header =
+    { $players ->
+        [one] {$players} player on a watchlist has
+        *[other] {$players} players on a watchlist have
+    } connected to {$serverName}
+
+discord-watchlist-connection-entry = - {$playerName}
+
+discord-watchlist-connection-no-expiry = (permanent)
+discord-watchlist-connection-expiry = (expires <t:{$expiry}:R>)
+
+discord-watchlist-connection-message = with message "{$message}"
+
+discord-watchlist-connection-more = (and { $watchlists ->
+        [one] {$watchlists} other watchlist
+        *[other] {$watchlists} other watchlists
+    })
```

Comment:
Concatenating messages like this is not the correct way to use Fluent, this is impossible to localize to most languages. You should try to keep these messages kept-together as much as possible.